Mathlib/Algebra/Order/Monoid/Defs.lean

```diff
@@ -46,43 +46,43 @@ attribute [to_additive] OrderedCommMonoid
 section OrderedInstances
 
 @[to_additive]
-instance OrderedCommMonoid.to_CovariantClass_left (M : Type _) [OrderedCommMonoid M] :
+instance OrderedCommMonoid.toCovariantClassLeft (M : Type _) [OrderedCommMonoid M] :
```

Comment:
If I understand correctly, either `to_covariantClass_left` or `toCovariantClass_left` would be correct (depending on which way we decide for `to`), but the `_left` should be snake-cased since it's not part of the structure name.

Reply:
Thanks. I had overlooked that `CovariantClass` is a `Prop`.